src/MediaWiki/Collator.php

```
@@ -70,7 +63,7 @@ public static function singleton( $collationName = '' ) {
 	 *
 	 * @return string
 	 */
-	public function armor( $text, $source = '' ) {
+	public function armor( string $text, $source = '' ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$source'. (undefined)

(UnusedFormalParameter)